.gitignore

```diff
@@ -8,7 +8,6 @@ doc.json
 typemanifest.txt
 /benchmarks/*.png.png
 
-Refresh (Linux).sh
 Validate (Linux).sh
-Refresh (macOS).command
 Validate (macOS).command
+# CI needs the “Refresh” scripts.
```

Comment:
Why can’t `refresh` and `validate` just be run from the Workspace executable?

Reply:
They can, but you need a way to get the Workspace executable onto Travis CI in the first place.

The purpose of the scripts is to retrieve the executable if needed before running it, and in tandem with the generated travis setup, they use caching to speed up subsequent jobs.

You could do all that manually instead if you want. Workspace just generates the scripts so that users don’t have to craft those steps themselves unless they want to.